.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - '6'
```

Comment:
we should keep support for node `6` and above.

Reply:
Not sure why need keep support since node 6? [node 6 is an unmaintained](https://nodejs.org/en/about/releases/) right now.

In my opinion, we should support node versions from latest active LTS (node v10).

Reply:
Okay. Agree, https://github.com/workshopper/org/blob/master/COMPATIBILITY.md

Not sure about 8 though. @martinheidegger ?

Reply:
According the example in the compatibility file that you linked, we should support v8 too.

But I'm not sure that someone encourage new users to use that version to learn node.

Anyway, I'll update the PR adding v8.

Reply:
We can drop Node 8 based on my reading of that doc since it's an LTS release that is older than a year. It's LTS phase started in 2017-10-31. I'm dropping Node 8 in this PR: https://github.com/workshopper/learnyounode/pull/677 since it doesn't support the `URL` class and we need to start recommending that since `require('url')` is deprecated.